okhttp/src/main/java/okhttp3/Cache.java

```diff
@@ -175,6 +175,12 @@
   private int hitCount;
   private int requestCount;
 
+  /**
+   * Create a cache which will reside in {@code directory}.
+   *
+   * @param directory a writable directory
+   * @param maxSize the maximum number of bytes this cache should use to store
```

Comment:
If you add `{@code maxSize}` in the sentence above you can drop these two `@param` tags. Maybe `Create a cache of at most {@code maxSize} bytes in {@code directory}.`?

Reply:
Sounds good! Updated